gemrb/core/GameScript/GSUtils.cpp

```diff
@@ -485,6 +485,11 @@ void DisplayStringCore(Scriptable* const Sender, int Strref, int flags)
 			strlcpy(Sound, sb.Sound, sizeof(Sound));
 		}
 		if (sb.text) {
+			ieDword charactersubtitles = 0;
+			core->GetDictionary()->Lookup("Subtitles", charactersubtitles);
+			if (!charactersubtitles) {
+				flags &= ~DS_CONSOLE;
+			}
```

Comment:
It has been looked up already around line 462, so why not just toggle it there? Did you find some other use than what was mentioned in the bug that requires us to always disable?

Reply:
It never enters the if https://github.com/gemrb/gemrb/blob/b94b3069cb09fd4aa49ddc1526c177131432c12a/gemrb/core/GameScript/GSUtils.cpp#L438
So I put it when it was showing the text.

Reply:
What are you testing with? Both idle sounds and selection sounds are verbal constants.

Reply:
I have followed the steps:
Steps to reproduce

    Disable subtitles
    Stand around make a coffee
    Check chat log

Reply:
Checked the code and I see, it depends on what kind of actor is running it. Still, there's no need to look up the configuration twice. And you can do the actual check in the same block that disables it for PST.

Reply:
Change to put it on the upper if of PST.

Reply:
You can move it up further, so the lookup at 462 isn't needed.

Reply:
@lynxlynxlynx move it above the if for be called only once.